install-load-test-server.md

```diff
@@ -0,0 +1,68 @@
+# Installing the `loadtest` program
+
+Before downloading the source and building the `loadtest` program, you must install a few prerequisites. If your machine is already set up for building Mattermost Server, you can skip to Step 5 where you install Glide.
+
+1. Install the build-essential package.
+
+    `sudo apt-get install build-essential`
+
+1. If you're installing `loadtest` on Linux: Download and install Go 1.8 for Linux.
+
+    a. Download the Go binary.
+
+      `wget https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz`
+
+    b. Install the Go binary.
+
+      `sudo tar -C /usr/local -xzf go1.8.linux-amd64.tar.gz`
+
+    c. Modify permissions on `/usr/local/go`. Replace {user} and {group} with the user and group that you are logged in as.
```

Comment:
You have to extract into `/usr/local` as root because regular users don't have permissions for that. But then the permissions on the go executable are for root, which means you can't use go without first changing the owner and group.